code/datums/mind.dm

```diff
@@ -101,6 +101,10 @@
 	martial_art = default_martial_art
 	init_known_skills()
 	set_assigned_role(SSjob.GetJobType(/datum/job/unassigned)) // Unassigned by default.
+	//SKYRAT EDIT START - OPFOR SYSTEM
+	SSopposing_force.give_opfor_datum(src)
```

Comment:
It does, right here

Reply:
Currently, that just gives them a datum, not generate an application. It gives them the datum so that they can get the action button ([modular_skyrat/modules/opposing_force/code/opposing_force_datum.dm](https://github.com/Skyrat-SS13/Skyrat-tg/pull/10994/files#diff-62f3ff39330dfbc0452166aca7d1a5662957605f87270ea880b77895c343c5dc) line 850), I'll take a second look at this tomorrow if there's anything I could do differently, but I do recall feeling fairly confident about it

Reply:
Changed